catalog/hyperledger/catalog.bom

```diff
@@ -125,7 +125,7 @@ brooklyn.catalog:
         name: "hyperledger-membersrvc-pod"
 
         brooklyn.config:
-          resource: "https://raw.githubusercontent.com/cloudsoft/kubernetes-hyperledger/master/hyperledger-membersrvc-pod.yaml"
+          resource: "classpath://io.brooklyn.hyperledger:hyperledger/hyperledger-membersrvc-pod.yaml"
```

Comment:
Change this, and other references to `classpath://io.brooklyn.hyperledger.kubernetes:hyperledger/hyperledger-membersrvc-pod.yaml`